e2e/test/scenarios/dashboard-cards/reproductions/22265-broken-added-series-prevents-editing-card.cy.spec.js

```diff
@@ -68,18 +62,21 @@ describe("issue 22265", () => {
     cy.button("Save").click();
     cy.button("Saving…").should("not.exist");
 
+    cy.log("Update the added series' question so that it's broken");
+    const questionDetailUpdate = {
+      dataset_query: {
+        type: "native",
+        native: {
+          query: "SELECT --2",
+          "template-tags": {},
+        },
+        database: 1,
+      },
+    };
     cy.get("@invalidQuestionId").then(invalidQuestionId => {
-      visitQuestion(invalidQuestionId);
-      cy.icon("expand").click();
-      cy.get(".ace_content").type("{backspace}--2");
-      cy.findByTestId("native-query-editor-sidebar")
-        .button("Get Answer")
-        .click();
-      cy.findByText("Save").click();
+      cy.request("PUT", `/api/card/${invalidQuestionId}`, questionDetailUpdate);
```

Comment:
I ran the test for each variation 5 times. And using API reduces the test time from about 7s to 4s